Doc/library/code.rst

```diff
@@ -68,17 +68,14 @@ build applications which provide an interactive interpreter prompt.
 Overriding Console Output
 -------------------------
 
-To customize console output a user can replace :func:`sys.displayhook`
-and/or :func:`sys.excepthook`. Alternatively, users can override
-:meth:`InteractiveConsole.write` in a derived class.
-
-The destination of console output is controlled as follows:
-
-* The return value from successfully interpreted Python statements are printed
-  by calling :func:`sys.displayhook`. This is a side effect of
-  :func:`compile_command` passing ``'single'`` for *symbol*.
-* Exception tracebacks and syntax errors are written to :data:`sys.stderr` by 
-  default but can be redirected by setting :func:`sys.excepthook`.
+All output is printed to :data:`sys.stderr` by default.  A user can override 
+:meth:`InteractiveConsole.write` in a derived class to change that.  Alternatively, 
```

Comment:
Please wrap this so that no line is more than 80 chars.

Before we make this documented behavior, let's make sure that there are unit tests ensuring it is true.

Reply:
I think there are tests for excepthook, but not for displayhook:
https://github.com/python/cpython/blob/main/Lib/test/test_code_module.py